src/Equinox.Projection.Kafka/Equinox.Projection.Kafka.fsproj

```diff
@@ -21,6 +21,7 @@
     <PackageReference Include="FSharp.Core" Version="3.1.2.5" Condition=" '$(TargetFramework)' != 'netstandard2.0' " />
     <PackageReference Include="FSharp.Core" Version="4.3.4" Condition=" '$(TargetFramework)' == 'netstandard2.0' " />
 
+    <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
```

Comment:
ideally this ref would not go here if possible - either we use serilog to shred datastructures down or it lives outside if possible